<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QWidget" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="KLineEdit" name="wordEntry"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Pronunciation</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>pronunciationEntry</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="KLineEdit" name="pronunciationEntry"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Example</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>exampleEntry</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="KLineEdit" name="exampleEntry"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Paraphrase</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>paraphraseEntry</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="KLineEdit" name="paraphraseEntry"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Comment</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>commentEntry</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="KLineEdit" name="commentEntry"/>
     </item>
     <item row="1" column="2">
      <widget class="KLineEdit" name="translationEntry"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="languageLabel">
       <property name="text">
        <string>Language</string>
       </property>
       <property name="buddy">
        <cstring>wordEntry</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="secondLanguageLabel">
       <property name="text">
        <string>Language</string>
       </property>
       <property name="buddy">
        <cstring>wordEntry</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Lesson</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lessonLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Word type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="Editor::WordTypeView" name="wordTypeView"/>
     </item>
     <item row="3" column="2">
      <widget class="KLineEdit" name="pronunciationEntry_2"/>
     </item>
     <item row="4" column="2">
      <widget class="KLineEdit" name="exampleEntry_2"/>
     </item>
     <item row="5" column="2">
      <widget class="KLineEdit" name="paraphraseEntry_2"/>
     </item>
     <item row="6" column="2">
      <widget class="KLineEdit" name="commentEntry_2"/>
     </item>
     <item row="7" column="2">
      <widget class="Editor::WordTypeView" name="wordTypeView_2"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KPushButton" name="previousButton">
       <property name="text">
        <string>&amp;Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="currentIndex">
       <property name="text">
        <string/>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="nextButton">
       <property name="text">
        <string>&amp;Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="newButton">
       <property name="text">
        <string>&amp;Add New</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Editor::WordTypeView</class>
   <extends>QTreeView</extends>
   <header>vocabulary/wordtypeview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>wordEntry</tabstop>
  <tabstop>translationEntry</tabstop>
  <tabstop>pronunciationEntry</tabstop>
  <tabstop>pronunciationEntry_2</tabstop>
  <tabstop>exampleEntry</tabstop>
  <tabstop>exampleEntry_2</tabstop>
  <tabstop>paraphraseEntry</tabstop>
  <tabstop>paraphraseEntry_2</tabstop>
  <tabstop>commentEntry</tabstop>
  <tabstop>commentEntry_2</tabstop>
  <tabstop>wordTypeView</tabstop>
  <tabstop>wordTypeView_2</tabstop>
  <tabstop>previousButton</tabstop>
  <tabstop>currentIndex</tabstop>
  <tabstop>nextButton</tabstop>
  <tabstop>newButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
